CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+## Contributing
+
+The book is in beta phase now. The following type of contributions are welcome:
+* Corrections to articles.
+* Contributions to the Chinese translation. Please note that the [Chinese translation version](https://github.com/Golang101/golang101) is almost done.
+* Translations to other languages are welcome.
+
+Note: Please read [UPDATES.md](UPDATES.md) for the update history.
+
```

Comment:
Thanks for the improvements for the project.
I really would accept them (and another README one),
but I also feel this project is so small that there will not much contents to be put in CONTRIBUTING.md. 
And I think contribution info is important and deserved to be put on the first page.
Maybe a separated CONTRIBUTING.md file will become needed when the project/communication becomes larger later (if it is possible).

And although I'm also maintaining the Chinese translation version, I don't expect to maintain other translation versions. And currently, I also don't plan to make go101.org become a multilingual website. The Chinese version will also be put on another website. That means the Chinese version and the English version are two separated projects. 

The README.md file does need to be improved. I suggest modifying the Contributing section as:

```
Welcome to improve Go 101 by 
* submitting corrections for all kinds of mistakes, such as typos, grammar errors, wording inaccuracies, description flaws, code bugs and broken links. 
* suggestion interesting Go related contents.

Translations are also welcome. Here lists the ongoing translation projects:
*  [Chinese translation version](https://github.com/Golang101/golang101) (almost done).

Reply:
Hi, 

Thank you for the feedback. I really appreciate the suggestions. Based on your recommendation, I am closing this PR. I have made the above changes to the contributing section here: https://github.com/go101/go101/pull/107